app/models/invite.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Invite < ApplicationRecord
+  belongs_to :organization
+  belongs_to :membership, optional: true
+
+  INVITE_STATUS = %i[
+    pending
+    accepted
+    revoked
+  ].freeze
+
+  enum status: INVITE_STATUS
+
+  validates :email, email: true, presence: true
```

Comment:
Do we still need to validate the presence?
Is it not already done on the email validator?

Reply:
the email validator should do the job yes! maybe a small spec around it could help? :D

Reply:
Indeed ! The validator seems to check the presence. I removed it

We have a test for that on the model's spec: https://github.com/getlago/lago-api/pull/438/files#diff-496401e9d70767dfb91ce412594c2603f37809f6ee69c90b5be808c30241a73aR30